ui/browser/browser-util.js

```diff
@@ -71,3 +71,13 @@ export function getWebView(document, pageIndex) {
   }
   return element.webview;
 }
+
+/**
+ * Takes an object and attaches a unique `id` to it.
+ *
+ * @TODO Should we start using UUIDs?
```

Comment:
Incremental ids are unique anyway, as long as no other part of the application has a different generator. Could pull in some UUID library though, especially if we're saving things to databases where we truly want unique ids, there's tons on npm and supposedly pretty fast.